### PR TITLE
[VET TEC] Update bank account helper text

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Update the helper text of the VET TEC bank account info page.

## Testing done
Tested locally

## Screenshots
image

## Acceptance criteria
- [x] On the Banking Information page (aka Personal Information page), the helper text has been expanded to apply to those who do not have OR do not want to provide their bank account information.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]
LGTM

**Comment by dmccurdybah** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/edu-benefits/0994/content/bankInformation.jsx [hunk 1]
```diff
@@ -26,14 +26,22 @@ export const bankInfoHelpText = (
...
-    you may have.
+    <p>
+      The Department of Treasury requires all federal benefit payments be made
+      by electronic funds transfer (EFT), also called direct deposit.
+    </p>
+    <p>
+      If you don’t have a bank account, or don’t wish to provide your bank
+      account information, you must receive your payment through Direct Express
+      Debit MasterCard. To request a Direct Express Debit MasterCard, apply at{' '}
+      <a href="www.usdirectexpress.com">www.usdirectexpress.com</a> or call
```
Comment (jbalboni) [inline src/applications/edu-benefits/0994/content/bankInformation.jsx]: Should this open in a new tab?

**Review by jbalboni — APPROVED** [review]
One question, but it looks good otherwise.

File: src/applications/edu-benefits/0994/content/bankInformation.jsx (hunk 1 above)
Comment (dmccurdybah) [reply src/applications/edu-benefits/0994/content/bankInformation.jsx]: Good question, thanks!

**Pull request merged**